doc/scenario.md

```diff
@@ -0,0 +1,317 @@
+- `instance_info` - information for a current instance. It's a dictionary with fields:
+  - `app_conf_file` - path to file with application config;
+  - `conf_file` - path to file with instance config;
+  - `instance_id` - ID of instance (e.g. for section name in config file);
+  - `console_sock` - path to control socket of instance;
+  - `pid_file` - path to pid file of instance;
+  - `work_dir` - path to working directory of instance;
+  - `tmpfiles_conf` - path to config file of temporary files;
+  - `dist_dir` - path to directory of distributed package;
+  - `instance_dist_dir` - path to instance link to distributed package;
+- `single_instances_for_each_machine` - list of instances (Ansible hosts), one for each physical machine. Can be used,
+  for example, in `delegate_to`;
+- `not_expelled_instance` - information about one not expelled instance. It's a dictionary with fields:
+  - `name` - instance name (Ansible host);
+  - `console_sock` - path to control socket of instance;
+- `scenario_steps` - description of scenario steps. It's a dictionary with fields:
+  - `name` - name of step;
+  - `path` - path to YAML file of step.
+
+## Role Steps Description
+
+### deliver_package
+
+Delivery of the application package to physical machines.
+
+Input facts (set by role):
+- `single_instances_for_each_machine` - list of instances (Ansible hosts), one for each physical machine.
+
+Input facts (set by config):
+- `cartridge_package_path` - path to file of package to delivery.
+
+Output facts:
+- `delivered_package_path` - remote path to file of delivered package.
+
+### update_package
+
+Install the delivered package on physical machines.
+
+Input facts (set by role):
+- `single_instances_for_each_machine` - list of instances (Ansible hosts), one for each physical machine;
+- `delivered_package_path` - remote path to file of delivered package.
+
+Input facts (set by config):
+- `instance_info` - information for a current instance ([more details here](#role-facts-descriptions));
+- `cartridge_app_name` - application name;
+- `cartridge_enable_tarantool_repo` - indicates if Tarantool repository should be enabled;
+- `cartridge_install_tarantool_for_tgz` - indicates if Tarantool should be enabled when use TGZ package;
+- `cartridge_multiversion` - indicates that multiple package versions is enabled;
+- `cartridge_configure_tmpfiles` - indicates that tmpfiles config should be configured;
+- `cartridge_configure_systemd_unit_files` - indicates that systemd Unit files should be configured;
+- `cartridge_app_user` - user which will own the links;
+- `cartridge_app_group` - group which will own the links;
+- `cartridge_run_dir` - path to directory of instances sockets;
+- `cartridge_data_dir` - path to directory of instances data;
+- `cartridge_conf_dir` - path to directory of instances application configs;
+- `cartridge_install_dir` - path to directory with packages;
+- `cartridge_instances_dir` - path to directory with instances links to packages.
```

Comment:
You can simply remove this parameters, I will explain it within TGZ + Multiversion doc (all since `cartridge_app_user `).

Reply:
In #192.